example/reviews/server/server.js

```diff
@@ -1,12 +0,0 @@
-const express = require('express');
```

Comment:
Hahaha good call on removing the example folder

Reply:
yeah !